src/components/ExpandableInfo.tsx

```diff
@@ -1,216 +1,162 @@
+      defaultIsOpen: forceOpen,
+    })
+
+    const chevronFlip = {
+      collapsed: {
+        rotate: 0,
+        transition: {
+          duration: 0.1,
+        },
       },
-    },
-    expanded: {
-      rotate: 180,
-      transition: {
-        duration: 0.4,
+      expanded: {
+        rotate: 180,
+        transition: {
+          duration: 0.4,
+        },
       },
-    },
-  }
-  const showHide = {
-    collapsed: {
-      display: "none",
-    },
-    expanded: {
-      display: "inline-block",
-    },
-  }
-  const fadeInOut = {
-    collapsed: {
-      opacity: 0,
-      transition: {
-        duration: 0.1,
-      },
-    },
-    expanded: {
-      opacity: 1,
-      transition: {
-        duration: 0.4,
-      },
-    },
-  }
-  return (
-    <Card background={background} className={className}>
-      <Content>
-        {image && <GatsbyImage image={image} alt="" />}
-        <TitleContent>
-          <Question>
-            <Header>
-              <Title>{title}</Title>
-            </Header>
-            <TextPreview>{contentPreview}</TextPreview>
-          </Question>
-        </TitleContent>
-        {!forceOpen && (
-          <ButtonContainer
-            variants={chevronFlip}
-            animate={isVisible ? "expanded" : "collapsed"}
-            initial={false}
-            onClick={() => setIsVisible(!isVisible)}
-          >
-            <Icon name="chevronDown" size="36" />
-          </ButtonContainer>
-        )}
-      </Content>
-      <motion.div
-        variants={expandCollapse}
-        animate={isVisible ? "expanded" : "collapsed"}
-        initial={false}
+    }
+
+    const animateToggle = isOpen ? "expanded" : "collapsed"
+
+    return (
+      <VStack
+        border="1px solid"
+        borderColor="border"
+        borderRadius="2px"
+        p={6}
+        mb={4}
+        spacing="0"
+        background={background ?? "background"}
+        position="relative"
+        _hover={{
+          "& img": {
+            transform: "scale(1.08)",
+            transition: "transform 0.1s",
+          },
+        }}
+        ref={ref}
+        {...rest}
       >
-        <motion.div
-          variants={showHide}
-          animate={isVisible ? "expanded" : "collapsed"}
+        <Stack
+          justify="space-between"
+          align="center"
+          gap={{ base: 8, md: 12 }}
+          flexDirection={{ base: "column", md: "row" }}
+          width="full"
+        >
+          {image && <GatsbyImage image={image} alt="" />}
+          <HStack gap={12} width="full">
+            <Box mr={4}>
+              <HStack
+                width="full"
+                my={4}
+                sx={{
+                  img: {
+                    mr: 6,
+                  },
+                }}
+              >
+                <Heading
+                  mt="0"
+                  mb={1}
+                  fontSize={{ base: "2xl", md: "2rem" }}
```

Comment:
If you replaced `md: "2rem"` with `md: 8` here, it will render `8px` because the fontSize prop does not like it.

Reply:
Oh got it. My bad haha thought it was a spacing prop 🤪